x/storage/keeper/keeper.go

```diff
@@ -1441,12 +1447,16 @@ func (k Keeper) RenewGroupMember(ctx sdk.Context, operator sdk.AccAddress, group
 	eventMembersDetail := make([]*types.EventGroupMemberDetail, 0, len(opts.Members))
 	for i := range opts.Members {
 		member := opts.Members[i]
-		memberExpiration := opts.MembersExpiration[i].UTC()
 		memberAcc, err := sdk.AccAddressFromHexUnsafe(member)
 		if err != nil {
 			return err
 		}
 
+		var memberExpiration *time.Time
+		if opts.MembersExpiration[i] != nil {
+			memberExpiration = opts.MembersExpiration[i]
```

Comment:
it is no difference to use `opts.MembersExpiration[i]` directly.

Reply:
Fixed